docs/src/main/asciidoc/spring-cloud-circuitbreaker-resilience4j.adoc

```diff
@@ -89,3 +89,10 @@ public Customizer<ReactiveResilience4JCircuitBreakerFactory> slowCusomtizer() {
 }
 ----
 ====
+
+==== Collecting Metrics
+
+Spring Cloud Circuit Breaker Resilience4j includes auto-configuration to setup metrics collection as long as the right
+dependencies are on the classpath.  To enable metric collection you must include `org.springframework.boot:pring-boot-starter-actuator`,
+`io.micrometer:micrometer-core`, and `io.github.resilience4j:resilience4j-micrometer`.  For more information on the metrics that
```

Comment:
Doesn't actuator include micrometer already? maybe in parens '(Included by actuator)' or something.